lab1/README.md

```diff
@@ -152,4 +152,186 @@ func main() {
 
 Any other goroutine that tries to access the shared resource while the mutex is locked will be blocked until the mutex is unlocked.
 
-- [Source Code of Mutex in Go](https://go.dev/src/sync/mutex.go)
\ No newline at end of file
+- [Source Code of Mutex in Go](https://go.dev/src/sync/mutex.go)
+
+## Channel
+Goroutines run in the same address space, so access to shared memory must be synchronized. In Golang, Channel is a powerful concurrency primitive function used for passing data between different goroutines. It provides an effective communication mechanism that allows goroutines to safely exchange information without requiring additional synchronization mechanisms.
+
+To create a channel, you can use Golang's built-in **`make`** function to establish an integer-type channel. Once successfully created, you can send data to the channel and receive data in another goroutine.
+
+Channels can be used to receive and send data through the **`<-`** operator:
+```go
+var c chan int
+
+ch := make(chan int)
+cs := make(chan string)
+cf := make(chan interface{})
+
+ch <- v    // transmit v to channel ch
+v := <-ch  // receive data from channel 'ch'，and asign value to v
+
+close(ch) //close Channel
+```
+
+You can also read a channel with iteration (**`for range`**):
+```go=
+func main() {
+  ch := make(chan int)
+  go func() {
+    for i := 0; i < 10; i++ { //assign integers to ch
+      ch <- i
+    }
+    close(ch)
+  }()
+
+  for v := range ch { //receive datas from ch iteratively
+    fmt.Println(v)
+  }
+}
+```
```

Comment:
Add description for:
- buffered channel/unbuffered channel
- one-way channel, ref: https://stackoverflow.com/questions/13596186/whats-the-point-of-one-way-channels-in-go

Reply:
Please also consider to non-blocking channel read operation (combine channel with `select`)